main.go

```diff
@@ -343,8 +345,15 @@ func (a *App) uploadToFlexera() {
 		}
 		if err != nil {
 			log.Println(err)
+			atLeastOneError = true
```

Comment:
We need to add the same atLeastOneError = true for all err validations, after validate err in StartBillUploadProcess and also after UploadFile

Reply:
Fixed.